doc/architecture/decisions/0003-store-local-copy-of-organisation-data.md

```diff
@@ -0,0 +1,26 @@
+# 3. Store local copy of Organisation Data
```

Comment:
Rather than `Store` is this about creating/generating a file containing all of the data?